### PR TITLE
feat(unlock-protocol-com) - use footer component

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

Replace the footer component.

### Demo
https://user-images.githubusercontent.com/20865711/207355900-b2e57f58-965e-46a8-a139-ee573634a103.mov

### Desktop
Screenshot 2022-12-13 at 15 06 57

### Mobile
Screenshot 2022-12-13 at 15 07 43

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/src/components/layout/DefaultLayout.tsx [hunk 1]
```diff
@@ -1,17 +1,119 @@
...
+        await fetch(endpoint, options)
+      },
+    },
+    logoUrl: 'https://unlock-protocol.com/',
+    privacyUrl: '/privacy',
+    termsUrl: '/terms',
+    actions: [
+      {
+        label: 'Launch App',
+        url: 'https://staging-app.unlock-protocol.com/locks',
```
Comment (julien51) [inline unlock-protocol-com/src/components/layout/DefaultLayout.tsx:44]: Not right! Please use env vars here to distinguish links between staging and prod

**Review by julien51 — CHANGES_REQUESTED** [review]
Link to the app is not correct :)

File: unlock-protocol-com/src/components/interface/Footer/index.tsx [hunk 2]
```diff
@@ -1,136 +0,0 @@
...
-} from '../../../config/constants'
-import { SOCIAL_URL } from '../../../config/seo'
-import { unlockConfig } from '../../../config/unlock'
-type LinkType = Record<'name' | 'href', string>
-
-const FOOTER_BOTTOM_NAVIGATION: Record<string, LinkType[]> = {
-  App: [
-    {
-      name: 'Creator Dashboard',
-      href: `${unlockConfig.appURL!}/locks`,
```
Comment (julien51) [inline unlock-protocol-com/src/components/interface/Footer/index.tsx:16]: See here @kalidiagne

**Comment by kalidiagne** [comment]
need https://github.com/unlock-protocol/unlock/pull/10575 before merge!

**Comment by kalidiagne** [comment]
> need #10575 before merge!

merged!

**Comment by kalidiagne** [comment]
> julien51

done

**Review by julien51 — APPROVED** [review]

**Pull request merged**